_archive/mv2/api/nativeMessaging/host/native-messaging-example-host

```diff
@@ -1,19 +1,14 @@
-#!/usr/bin/env python
-# Copyright (c) 2012 The Chromium Authors. All rights reserved.
```

Comment:
For this portion, keep the license and adjust the env to `#!/usr/bin/python3` to ensure that Chrome launches  this file with Python3, otherwise it will likely fail as it doesn't know what the file should be ran as.